app/models/marc_fields/bound_with_note.rb

```diff
@@ -2,22 +2,17 @@
 #  MarcBoundWith class to return 590 fields that have a $c
 ###
 class BoundWithNote < MarcField
-  def values
-    extracted_fields.map do |field, subfields|
-      id = subfields.find { |subfield| subfield.code == 'c' }.value[/^(\d+)/]
+  include ActionView::Helpers::UrlHelper
 
-      { id: id, value: display_value(field, subfields.reject { |subfield| subfield.code == 'c' }) }
-    end
-  end
+  private
 
-  def to_partial_path
-    'marc_fields/bound_with_note'
-  end
+  def subfield_value(field, subfield)
+    return super unless subfield.code == 'c'
```

Comment:
Specialize behavior only when the subfield code is `c`. Consistent with class comments above, so that's good.